src/common/utils/formatDateForChat.ts

```diff
@@ -0,0 +1,30 @@
+import { format } from 'date-fns';
+import { t } from 'i18next';
+
+function formatDateForChatList(date: Date): string {
+  const now = new Date();
+  const hours = 24;
+  const seconds = 3600;
+  const milliseconds = 1000;
+  const daysInAYear = 365;
+  const oneDayDifference = 1;
+  const twoDayDifference = 2;
+  const millisecondsInADay = hours * seconds * milliseconds;
+  const diff = (now.getTime() - date.getTime()) / millisecondsInADay;
+
+  const timeFormat = 'p';
+  const monthAndDayFormat = 'MMMM d';
+  const yearMonthAndDayFormat = 'yyyy MM d';
```

Comment:
it should be constand date format through full project. please check general constant file

Reply:
Added this formats to constant file